rs/cycles_account_manager/src/lib.rs

```diff
@@ -44,7 +44,7 @@ const SECONDS_PER_DAY: u128 = 24 * 60 * 60;
 
 /// Maximum payload size of a management call to update_settings
 /// overriding the canister's freezing threshold.
-const MAX_DELAYED_INGRESS_COST_PAYLOAD_SIZE: usize = 256;
+const MAX_DELAYED_INGRESS_COST_PAYLOAD_SIZE: usize = 267;
```

Comment:
This needed to be updated since additional field wasm_memory_threshold: Option<candid::Nat> is added to CanisterSettingArgs and its size is 11 Bytes.